sample_data/contiguous_usa_by_county_since_2001/usa_population_2021.csv

```diff
@@ -0,0 +1,3109 @@
+AFFGEOID,Cartogram Data (eg. Population),Color,Inset,Label,ALAND,AWATER,COUNTYNS,GEOID,ShapeName
+0500000US01001,"58,805",,,,1539634184,25674812,161526,1001,Autauga
```

Comment:
The key used for counties is "AFFGEOID" which does not give any meaningful information. Could you regenerate the CSV with "NAMELSAD" as the key for the GeoDiv name?

The "Cartogram Data (eg. Population)" column seems to have numbers wrapped with quotation marks. It is best to remove them since our code is unable to parse the data.

Reply:
Using NAMELSAD as the key for the GeoDiv doesn't work, as the counties have repeated names across different states (e.g. there are 31 Washington Counties, and 26 Jefferson Counties.) The only unique identifiers present are GEOIDs and AFFGEOIDs. Maybe we can discuss what to do with this issue in particular at the next meeting.